plutarch-test/plutarch-base/Plutarch/ApiSpec.hs

```diff
@@ -202,3 +213,54 @@ getFields = phoistAcyclic $ plam $ \addr -> psndBuiltin #$ pasConstr # addr
 
 dummyCurrency :: CurrencySymbol
 dummyCurrency = Value.currencySymbol "\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11"
+
+------------------- Mocking a ScriptContext ----------------------------------------
+
+validContext0 :: Term s PScriptContext
```

Comment:
I think these mocks should be put into a common module that's not a `*Spec`

Reply:
Yes, I will do this next, but I want to do it in a separate PR (#433) so as to keep this one small and scoped.